### PR TITLE
feat(onboarding): generate token v1

### PR DESCRIPTION
**Pull request by oliviertassinari**
Prepare #165

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -102,7 +102,7 @@
     "react-hot-loader": "3.0.0-beta.6",
     "react-redux": "^5.0.4",
     "react-router": "^3.0.2",
-    "recompact": "^2.2.4",
+    "recompact": "^2.3.1",
```
Comment (neoziro) [inline package.json]: Is it needed in `feat(onboarding): generate token v1`? #UnitaryCommitsPlease

File: seeds/screenshotDiffs.js [hunk 2]
```diff
@@ -1,92 +1,94 @@
...
-        baseScreenshotId: 9,
-        compareScreenshotId: 9,
-        score: 0,
-        jobStatus: 'complete',
-        validationStatus: 'unknown',
-        s3Id: null,
-        createdAt: new Date().toISOString(),
-        updatedAt: new Date().toISOString(),
-      }),
+      knex('screenshot_diffs').insert([
```
Comment (neoziro) [inline seeds/screenshotDiffs.js]: Same, is it needed for the feature?

File: src/config.js [hunk 3]
```diff
@@ -8,7 +8,6 @@ const config = convict({
       'production',
       'development',
       'test',
-      'browser.development',
```
Comment (neoziro) [inline src/config.js]: 👍  we shouldn't have other environment than production, dev, and test!

File: src/modules/components/WatchTaskContainer.js [hunk 4]
```diff
@@ -7,12 +7,18 @@ import Layout from 'material-ui/Layout'
 const styleSheet = createStyleSheet('WatchTaskContainer', theme => ({
   root: {
     padding: theme.spacing.unit,
+    minHeight: 76,
```
Comment (neoziro) [inline src/modules/components/WatchTaskContainer.js]: magic 😆

File: src/review/routes/profile/Profile.js [hunk 5]
```diff
@@ -22,10 +22,12 @@ import LayoutBody from 'modules/components/LayoutBody'
 import ReviewAppBar from 'review/modules/AppBar/AppBar'
 import actionTypes from 'review/modules/redux/actionTypes'
 
+const size = 120
```
Comment (neoziro) [inline src/review/routes/profile/Profile.js]: SIZE?

File: src/server/routes/api.js [hunk 6]
```diff
@@ -100,8 +100,9 @@ router.post('/builds', upload.array('screenshots[]', 500), errorChecking(
     )
 
     await buildJob.push(build.id)
-
-    res.send(build)
+    res.send({
```
Comment (neoziro) [inline src/server/routes/api.js]: `res.send({ build })`, prettier 🙏

File: package.json (hunk 1 above)
Comment (oliviertassinari) [inline package.json]: No it's not. I have an higher velocity by doing everything in a single commit

File: seeds/screenshotDiffs.js (hunk 2 above)
Comment (oliviertassinari) [inline seeds/screenshotDiffs.js]: Yes, it's preparing the seed.

Comment (oliviertassinari) [inline seeds/screenshotDiffs.js]: Well, it's a simplification here.

File: src/modules/components/WatchTaskContainer.js (hunk 4 above)
Comment (oliviertassinari) [inline src/modules/components/WatchTaskContainer.js]: Yes, it's. It's removing the jitter.

File: src/review/routes/profile/Profile.js (hunk 5 above)
Comment (oliviertassinari) [inline src/review/routes/profile/Profile.js]: Sure

File: src/server/routes/api.js (hunk 6 above)
Comment (oliviertassinari) [inline src/server/routes/api.js]: Fixing it.
At some point, I want to send the url to the interface.

**Pull request merged**